internal/explorer/explorer_utils.go

```diff
@@ -0,0 +1,31 @@
+package explorer
+
+import high "github.com/pb33f/libopenapi/datamodel/high/v3"
+
+func mergeParameters(commonParameters []*high.Parameter, operation *high.Operation) []*high.Parameter {
+	mergedParameters := commonParameters
```

Comment:
If the intention here is to clone the slice, you can use the built-in copy function:

```go
mergedParameters := make([]*high.Parameter, len(commonParameters))
copy(mergedParameters, commonParameters)
```